examples/helia-browser-verified-fetch/test/blockstore.js

```diff
@@ -0,0 +1,52 @@
+import { CodeError } from '@libp2p/interface'
+import { base32 } from 'multiformats/bases/base32'
+import { CID } from 'multiformats/cid'
+import * as raw from 'multiformats/codecs/raw'
+import * as Digest from 'multiformats/hashes/digest'
+import fixtures from './fixtures.js'
+
+export class MemoryBlockstore {
```

Comment:
only non-nit:

why do we need this custom memory store again? could we add a link to the comments/discussion around this and some explainers in the jsdoc?

can we `export class MemoryBlockstore extends MemoryBlockstoreOriginal` so we can remove a lot of this code that detracts from the example?

Reply:
Great idea! In fact, there's a lot of code duplication there.